packages/web/components/cards/stake-dashboard.tsx

```diff
@@ -77,11 +87,13 @@ export const StakeDashboard: React.FC<{
             title={t("stake.collectRewards")}
             titleIconUrl="www.google.com"
             tooltipContent="... placeholder content 1 ..."
+            onClick={collectRewards}
           />
           <RewardsCard
             title={t("stake.investRewards")}
             titleIconUrl="www.google.com"
             tooltipContent="... placeholder content 2 ..."
```

Comment:
While we're here, could we remove the google URL and placeholder content so it doesn't get forgotten?